src/Saas.IaC/keyVault.bicep

```diff
@@ -21,6 +21,12 @@ param azureAdB2cSignupAdminDomainSecretName string = 'signupadmin-AzureAdB2C--Do
 @description('The value of the Azure AD B2C Domain Key Vault Secret.')
 param azureAdB2cDomainSecretValue string
 
+@description('The name of the Azure AD B2C SaaS App Client Id Key Vault Secret.')
+param azureAdB2cSaasApplicationClientIdSecretName string = 'saasapplication-AzureAdB2C--ClientId'

```

Comment:
You also need to create a resource for these secrets with these parameters as the inputs. See line 102 below for an example.